packages/modular-scripts/src/react-scripts/config/getHttpsConfig.js

```diff
@@ -4,7 +4,6 @@ const fs = require('fs');
 const path = require('path');
 const crypto = require('crypto');
 const chalk = require('chalk');
-const paths = require('./paths');
```

Comment:
Paths are no longer a static js file but a TS script that's called only once per build and who's output is passed down to child functions that need it